src/structures.rs

```diff
@@ -90,6 +90,11 @@ pub enum SvNetType {
     IMPLICIT,
 }
 
+#[derive(Debug, Serialize, Clone)]
+pub struct SvPackedDimension {
```

Comment:
Can this be a type alias instead, then the YAML/JSON output won't need to say "dimension"?
Something like `type SvPackedDimension = (String, Option<String>);`

Reply:
Just removed dimension and implemented type alias.